src/seedu/addressbook/ui/TextUi.java

```diff
@@ -168,5 +169,17 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+    private static String getPrintableString(Printable... printables){
+    	StringBuilder printableString = new StringBuilder("");
+    	for(Printable p: printables){
+    		printableString.append(p.getPrintableString() + ", ");
+    	}
+    	return removeLastComma(printableString.toString());
+    }
```

Comment:
There is a way that you can combine the string without adding an additional comma, find out. :) Good job for putting this method at the right place, but you forget to write the comment.